### DISCUSSION TITLE
Is it possible to log in with new customer accounts when developing locally? (B2B Specifically)

### DISCUSSION THREAD

**Discussion by BrandonJamesBorders** [Shopify/dawn#3568] [Q&A]
I'm evaluating implementing B2B features in a theme and can't figure out how to log in. 

1. shopify theme dev --store={mystore}.myshopify.com
2. Open http://127.0.0.1:9292
3. Click the login link
4. Log in with the code emailed to me and I get sent to the {mystore.myshopify.com} URL. 

I could preview_theme_id query parameter and refresh after each change I suppose, but I'm hoping I'm just missing the proper process.

**Comment by dholland** [comment]
I also have this question. Did you find any solutions?
